src/core/core.tooltip.js

```diff
@@ -810,9 +811,13 @@ var exports = Element.extend({
 			ctx.fillStyle = textColor;
 			helpers.each(bodyItem.before, fillLineOfText);
 
-			helpers.each(bodyItem.lines, function(line) {
+			len = bodyItem.lines.length;
+			for (j = 0; j < len; j++) {
+				var line = bodyItem.lines[j];
```

Comment:
I wouldn't make a variable for this since it's only used once. Just do `fillLineOfText(lines[j]);`

Reply:
done